src/main/java/com/manardenza/servise/HotelService.java

```diff
@@ -41,6 +42,6 @@ public boolean cancelReservation(long roomId, long userId, long hotelId, Reserva
     }
 
     public List<Room> findRoom(Map<String, String> params) {
-        return null;
+        return new ArrayList<Room>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2293)